provider/datagen/src/transform/cldr/time_zones/convert.rs

```diff
@@ -444,47 +446,56 @@ fn metazone_periods_iter(
         Vec<MetaZoneForPeriod>,
         LiteMap<String, MetaZoneId>,
     ),
-) -> impl Iterator<Item = (TimeZoneBcp47Id, String, Option<MetaZoneId>)> {
+) -> impl Iterator<Item = (TimeZoneBcp47Id, i32, Option<MetaZoneId>)> {
     let (time_zone_key, periods, meta_zone_id_data) = pair;
     periods
         .into_iter()
         .map(move |period| match &period.uses_meta_zone.from {
             Some(from) => {
+                let parts: Vec<String> = from.split(' ').map(|s| s.to_string()).collect();
+                let date = &parts[0];
+                let time = &parts[1];
+                let date_parts: Vec<String> = date.split('-').map(|s| s.to_string()).collect();
+                let year = date_parts[0].parse::<i32>().unwrap();
+                let month = date_parts[1].parse::<u8>().unwrap();
+                let day = date_parts[2].parse::<u8>().unwrap();
+                let time_parts: Vec<String> = time.split(':').map(|s| s.to_string()).collect();
+                let hour = time_parts[0].parse::<i32>().unwrap();
+                let minute = time_parts[1].parse::<i32>().unwrap();
+                let iso = Date::new_iso_date(year, month, day).unwrap();
+                let minutes = Iso::unix_epoch_minute_from_iso(*iso.inner()) + hour * 60 + minute;
```

Comment:
Suggestion: Make a `DateTime<Iso>` instead of `Date<Iso>` and let it handle the hours and minutes for you.

Reply:
sg!